holmscan/webscan.py

```diff
@@ -15,6 +13,13 @@ class Webscan(HolmscanModule):
     def __init__(self, controller):
         super(Webscan, self).__init__(controller)
 
+    def get_web_scan(self, uuid):
+        url = "{0}/web-scans/{1}".format(
+            self.controller.conf.get("HOLMSEC_ENDPOINT"), uuid
```

Comment:
Alwasy have the habit of adding , to the last item in any place where you have lists of things. Also consider to put them on muliple lines, it always makes it easier to read.

Reply:
I am running [black](https://github.com/psf/black) to automatically format the code in order to avoid having to take stylistic decisions.

Black puts an ending `,` when it is splitting entries on separate lines. And it apparently does not do it when the entries are short enough to fit on one line. The reason probably has to do with how line-wise diffs would present themselves.